testNgIncludeGroup.xml

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
+<suite name="Including Groups Suite">
```

Comment:
@VictoriaBadenko please run this suite and check test results, what's wrong with it?
<img width="1659" alt="Screenshot 2023-08-03 at 1 49 49 PM" src="https://github.com/VictoriaBadenko/UnitTesting/assets/6151413/ad4e0f5c-48c4-4c03-8f51-dfe15fac9da2">

Reply:
Fixed this issue.